docs/config-file-glossary.rst

```diff
@@ -0,0 +1,92 @@
+####################
+Config file glossary
```

Comment:
My concern with config file documentation like this is that it decays over time because it gets forgotten. I understand the value of having it available in the documentation but that dimishes if it doesn't get updated with the options we support.

Would we be opposed to putting this information as comments in the config file itself?

Reply:
...Or somehow have this page be generated by something reading in the configuration file and its comments and outputting this page?

Reply:
I mean... *Technically* we could make a parser to read in the config file, and thus output this page. 

But... I think, since the `env.example` file doesn't get updated too often, we can keep the official documentation in the readthedocs. I'm not opposed to adding more comments into the `env.example` file in general, though. At least, maybe add a link to this readthedocs page instead of having `See the README for instructions.` at  the top of the file.

Reply:
@thenaterhood There might be a way to include the contents of a file. If we commented the example `env` file, we could use the file as living documentation. What do you think? We can open a new issue to track this.